datanode/gateway/graphql/update_referral_program_resolver.go

```diff
@@ -28,8 +28,9 @@ func (u updateReferralProgramResolver) BenefitTiers(_ context.Context, obj *vega
 	return obj.Changes.BenefitTiers, nil
 }
 
-func (u updateReferralProgramResolver) EndOfProgramTimestamp(_ context.Context, obj *vega.UpdateReferralProgram) (string, error) {
-	return time.Unix(obj.Changes.EndOfProgramTimestamp, 0).String(), nil
+func (u updateReferralProgramResolver) EndOfProgramTimestamp(_ context.Context, obj *vega.UpdateReferralProgram) (int64, error) {
+	endTime := time.Unix(obj.Changes.EndOfProgramTimestamp, 0)
+	return endTime.UnixNano(), nil
```

Comment:
Does this still return rfc3339? This is what the Timestamp scalar represents in the schema, and would be preferred.

Reply:
See https://github.com/vegaprotocol/vega/blob/develop/datanode/gateway/graphql/schema.graphql#L4969

Reply:
Yes this will be returning an rfc3339.